test/gtest/ucs/test_async.cc

```diff
@@ -379,6 +380,9 @@ class test_async_mt : public test_async {
         spawn();
         suspend();
 
+        // The checking of is_blocked works only when async mode is THREAD
+        // (spinlock or mutext), because we rely on is_bloceked which can
+        // be changed by spawned threads
```

Comment:
mutext->mutex
but is blocked still uses a counter right? are you sure it fails with other modes?
since other things in this function work, for example block()/unblock()

Reply:
> 2\. why would it fail? don't we already run some other tests with multiple threads?

we run, but this test blocks async in two threads simultaneously - poll_block and signal don't allow it

Reply:
i see, so pls add skip in `UCS_TEST_P(test_async_event_mt, check_blocks_multithread)` and write as a comment that "this test blocks async in two threads simultaneously - poll_block and signal don't allow it"

Reply:
done